examples/migrate_push_based_log_export_to_log_integration/README.md

```diff
@@ -0,0 +1,33 @@
+# Migration from `mongodbatlas_push_based_log_export` to `mongodbatlas_log_integration`
+
+This directory contains examples demonstrating how to migrate from `mongodbatlas_push_based_log_export` to `mongodbatlas_log_integration` using a **create-before-destroy** pattern. For more details, please refer to the [Migration Guide: Push-Based Log Export to Log Integration](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/push-based-log-export-to-log-integration-migration-guide).
+
+## Overview
+
+The `mongodbatlas_log_integration` resource provides enhanced log export capabilities compared to `mongodbatlas_push_based_log_export`:
+- Faster log export intervals (1 minute vs 5 minutes)
+- Configurable log types (`MONGOD`, `MONGOS`, `MONGOD_AUDIT`, `MONGOS_AUDIT`)
+- Optional AWS KMS key support for server-side encryption
+
+## Migration Approach
+
+Unlike migrations that use the `moved` block, this migration follows a **create-before-destroy** pattern:
+1. Create the new `mongodbatlas_log_integration` resource alongside the existing one
+2. Validate that logs are appearing at the new configuration
+3. Remove the old `mongodbatlas_push_based_log_export` resource
+
+This approach ensures continuous log delivery during migration without gaps.
+
+## Examples
+
+Please check the [basic/](./basic/README.md) folder. This shows a step-by-step migration through three phases:
```

Comment:
nit: Dont know if we are using the `basic` terminology in other migration examples, but if not would favour a more explict name like `direct-resource-usage`